src/backend/TrafficCourtsApi/Dockerfile.ticket-worker

```diff
@@ -0,0 +1,17 @@
+ARG DOTNET_SDK_VERSION=5.0
+ARG DOTNET_STARTUP_PROJECT=ticket-worker/ticket-worker.csproj
+ARG DOTNET_RUNTIME_BASE=registry.access.redhat.com/ubi8/dotnet-50-runtime
+
+FROM mcr.microsoft.com/dotnet/sdk:$DOTNET_SDK_VERSION AS build
+
+WORKDIR /src
+COPY . .
+RUN dotnet publish $DOTNET_STARTUP_PROJECT -c Release -o /opt/app-root/app
+
+FROM $DOTNET_RUNTIME_BASE AS base
+WORKDIR /opt/app-root/app
+EXPOSE 8080
+COPY --from=build /opt/app-root/app .
+
+ENV DOTNET_ASSEMBLY_NAME=ticket-worker.dll
+ENTRYPOINT ["sh","-c", "dotnet ${DOTNET_ASSEMBLY_NAME}"]
```

Comment:
I am confused why we run `sh -c dotnet ${DOTNET_ASSEMBLY_NAME}`  Doesn't just using `dotnet ${DOTNET_ASSEMBLY_NAME}` work?